guides/common/modules/ref_dhcp-isc-settings.adoc

```diff
@@ -0,0 +1,45 @@
+[id="DHCP_ISC_Settings_{context}"]
+= dhcp_isc Settings
+
+The _dhcp_isc_ provider uses a combination of the ISC DHCP server OMAPI management interface and parsing of configuration and lease files.
+This requires it to be run either on the same host as the DHCP server.
```

Comment:
There is an **or** missing for the **either**.

Reply:
Reworded.